Libraries/LibWeb/CSS/SelectorEngine.cpp

```diff
@@ -227,7 +227,9 @@ static inline Web::DOM::Attr const* get_optionally_namespaced_attribute(CSS::Sel
     }
 
     switch (namespace_type) {
-    case CSS::Selector::SimpleSelector::QualifiedName::NamespaceType::Default: // "In keeping with the Namespaces in the XML recommendation, default namespaces do not apply to attributes, therefore attribute selectors without a namespace component apply only to attributes that have no namespace (equivalent to "|attr")"
+    /* "In keeping with the Namespaces in the XML recommendation, default namespaces do not apply to attributes,
```

Comment:
Use `//` for comments. (I thought this was in `CodingStyle.md` but apparently not, that file needs updating...)